assets/src/scripts/components/Search.tsx

```diff
@@ -9,18 +9,31 @@ interface SearchProps {
 }
 
 export default function Search({ draftURL, searches }: SearchProps) {
+  const [activeUrl, setActiveUrl] = useState<string>(
+    () => searches.find((search) => search.active)?.url || "",
+  );
+
+  const handleClick = (url: string) => (e: React.MouseEvent) => {
+    // Don't trigger if clicking the Remove button
+    if (!(e.target as HTMLElement).closest("button")) {
+      setActiveUrl(url);
+    }
+  };
+
   return (
-    <>
-      <h3 className="h6">Searches</h3>
-      <ListGroup>
+    <Card>
+      <Card.Header as="h2" className="h6 font-weight-bold">
+        Previous searches
+      </Card.Header>
+      <ListGroup variant="flush">
         {searches.map((search) => (
           <React.Fragment key={search.url}>
             {search.delete_url ? (
               <ListGroup.Item
                 action
-                active={search.active}
-                className="py-1 px-2"
+                active={search.url === activeUrl}
                 href={search.url}
+                onClick={handleClick(search.url)}
               >
                 <Form action={search.delete_url} method="post">
```

Comment:
I'd like to get rid of the float for the button, so I'd recommend the following CSS:

```jsx
<Form
  action={search.delete_url}
  className="d-flex justify-content-between align-items-center"
  method="post"
>
```

```jsx
<Button
  aria-label="remove search"
  className="ml-2"
  name="delete-search"
  type="submit"
  size="sm"
  variant={
    search.url === activeUrl ? "light" : "outline-primary"
  }
>
```